mod/photos.php

```diff
@@ -1411,17 +1411,15 @@ function photos_content(App $a)
 		if (count($linked_items) && strlen($link_item['tag'])) {
 			$arr = explode(',', $link_item['tag']);
 			// parse tags and add links
-			$tag_str = '';
+			$tag_arr = [];
 			foreach ($arr as $t) {
-				if (strlen($tag_str)) {
-					$tag_str .= ', ';
-				}
-				$tag_str .= BBCode::convert($t);
+				array_push($tag_arr, ['name' => BBCode::convert($t),
+					'removeurl' => '/tagrm/'.$link_item['id'] . '/' . bin2hex($t)]);
 			}
-			$tags = [L10n::t('Tags: '), $tag_str];
+			$tags = ['title' => L10n::t('Tags: '), 'tags' => $tag_arr];
 			if ($cmd === 'edit') {
-				$tags[] = 'tagrm/' . $link_item['id'];
-				$tags[] = L10n::t('[Remove any tag]');
+				$tags += ['removeanyurl' => 'tagrm/' . $link_item['id']];
+				$tags += ['removetitle' => L10n::t('[Remove any tag]')];
```

Comment:
Shouldn't this be `Remove all tags`?

Reply:
The `tagrm` page as it is now (and I didn't change it function) also provides e.g. to only delete the first and third tag.
But since every tag has a remove button now the function could be changed. Did you mean this ?

Reply:
No, I just mean the wording. Is this link removing all of the photo tags?

Reply:
No, you're right the wording is wrong. The GET call to `tagrm/[id]/[tag as hex]` deletes just the provided tag.

Reply:
No it don't. `tagrm` provides a checkbox for every tag. On default no checkbox is selected. But it is possible to remove all tags. I guess that's the reason why someone had choose the term `Remov any tag` ?

Reply:
It probably should read "Select tags to remove" then.